pkg/config/api_config.go

```diff
@@ -27,8 +27,11 @@ import (
 
 import (
 	fc "github.com/dubbogo/dubbo-go-pixiu-filter/pkg/api/config"
+
 	etcdv3 "github.com/dubbogo/gost/database/kv/etcd/v3"
+
 	perrors "github.com/pkg/errors"
+
 	"go.etcd.io/etcd/api/v3/mvccpb"
 )
```

Comment:
Those blank line kind of strange, isn't it ?

Reply:
> Those blank line kind of strange, isn't it ?

+1

Reply:
Cai boss said follow the code formatting tool